data_diff/diff_tables.py

```diff
@@ -100,17 +101,32 @@ def __iter__(self):
             self.result_list.append(i)
             yield i
 
-    def _get_stats(self) -> DiffStats:
+    def _get_stats(self, is_dbt:bool = False) -> DiffStats:
         list(self)  # Consume the iterator into result_list, if we haven't already
 
+        key_columns = self.info_tree.info.tables[0].key_columns
         diff_by_key = {}
+        extra_column_diffs = None
+        if is_dbt:
+            extra_column_values_store = {}
+            extra_columns = self.info_tree.info.tables[0].extra_columns
+            extra_column_diffs = {k:0 for k in extra_columns}
+
         for sign, values in self.result_list:
-            k = values[: len(self.info_tree.info.tables[0].key_columns)]
+            k = values[: len(key_columns)]
+            if is_dbt:
+                extra_column_values = values[len(self.info_tree.info.tables[0].key_columns) :]
```

Comment:
```suggestion
                extra_column_values = values[len(key_columns) :]
```
I think this is equivalent

Reply:
If we're using `len(key_columns)` in a couple places we should consider a len_key_columns variable

Reply:
@dlawin made this change and created `len_key_columns` 👍